activemq-broker/src/main/java/org/apache/activemq/broker/jmx/AnnotatedMBean.java

```diff
@@ -191,10 +191,38 @@ public Object invoke(String s, Object[] objects, String[] strings) throws MBeanE
             entry.setUser(caller);
             entry.setTimestamp(System.currentTimeMillis());
             entry.setOperation(this.getMBeanInfo().getClassName() + "." + s);
-            entry.getParameters().put("arguments", objects);
+
+            try
+            {
+               if (objects.length == strings.length)
```

Comment:
Should we ensure the brackets conform to Egyptian Brackets as the style used by the rest of the source file?